internal/logger/writer_logger.go

```diff
@@ -15,6 +18,7 @@ func (l *WriterLogger) WriteLog(level Level, line string) {
 	if level < l.level {
 		return
 	}
+	line = fmt.Sprintf("%s %s", time.Since(startTime), line)
```

Comment:
New to go, but does this print the time since the `writer_logger` module has been loaded?